Userland/Libraries/LibDiff/Generator.cpp

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright (c) 2021, Mustafa Quraish <mustafa@cs.toronto.edu>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include "Generator.h"
+
+namespace Diff {
+
+Vector<Hunk> from_text(StringView const& old_text, StringView const& new_text)
+{
+    auto old_lines = old_text.lines();
+    auto new_lines = new_text.lines();
+
+    enum Direction {
```

Comment:
You are already using this like `enum class` so declare it like that?

Reply:
Didn't know there was a difference, TIL! Done.